app/src/marketplace/utils/product.js

```diff
@@ -58,6 +58,10 @@ export const mapProductFromApi = (product: Product | EditProduct): Product => {
     return {
         ...product,
         pricePerSecond,
+        pendingChanges: {
+            name: 'asdasd',
+            adminFee: 0.7,
+        },
```

Comment:
This data is for testing, the product data should be overlaid with the pending changes if the product is live and then cleared after publishing.